os/net/lwip/src/core/ipv4/ip_addr.c

```diff
@@ -53,9 +53,11 @@
  *
  */
 
+#include <arpa/inet.h>
 #include <net/lwip/opt.h>
 #include <net/lwip/ipv4/ip_addr.h>
 #include <net/lwip/netif.h>
+#include <net/lwip/debug.h>
```

Comment:
Are these header files necessary here? Please check again and ignore if found necessary.